docs/Zendesk-batchsource.md

```diff
@@ -3,7 +3,10 @@
 
 Description
 -----------
-This source reads objects from Zendesk.
+This source reads objects from Zendesk.It extracts reportable data from the Zendesk objects. 
+The Zendesk Batch Source plugin enables bulk data extraction from Zendesk. 
+You can configure and execute bulk data transfers from Zendesk without any coding
```

Comment:
missing period at the end of the sentence

Reply:
done